airflow-core/src/airflow/config_templates/airflow_local_settings.py

```diff
@@ -71,22 +71,25 @@
         },
     },
     "filters": {
-        "mask_secrets": {
-            "()": "airflow.sdk.execution_time.secrets_masker.SecretsMasker",
+        "mask_secrets_core": {
+            "()": "airflow._shared.secrets_masker.secrets_masker._secrets_masker",
+        },
+        "mask_secrets_sdk": {
+            "()": "airflow.sdk._shared.secrets_masker.secrets_masker._secrets_masker",
```

Comment:
Note to self: This is fine for this PR, but we're going to have to do something about this for 3.1, else this will fail to import the class if task-sdk isn't installed. I should address this as part of #52651

Reply:
Do we want to create an issue so we do not forget?

Reply:
>Do we want to create an issue so we do not forget?

Yes, please